src/DataDictionary/highlightHelper.jsx

```diff
@@ -0,0 +1,320 @@
+import React from 'react';
+import { getPropertyDescription, getType } from './utils';
+
+const escapeReturnChar = (str, newlineClassName) => {
+  if (!str) return str;
+  const pieces = str.split('\\n');
```

Comment:
are there meant to be two backslashes here?

Reply:
Some of the origin text have "\n", and we need another `\` to escape `\` itself, that's why there're two backslashes here